addons/binding/org.openhab.binding.evohome/ESH-INF/thing/bridge.xml

```diff
@@ -0,0 +1,37 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<thing:thing-descriptions bindingId="evohome" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns:thing="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0" xsi:schemaLocation="http://eclipse.org/smarthome/schemas/thing-description/v1.0.0 http://eclipse.org/smarthome/schemas/thing-description-1.0.0.xsd">
+	<!-- evohome Gateway Bridge -->
+	<bridge-type id="account">
+		<label>evohome Account</label>
+		<description>The evohome account is used to connect to your Total Connect Comfort (TCC) using your TCC username and password.</description>
+		<config-description>
+			<parameter-group name="auth">
+				<label>Authentication</label>
+				<description>Contains the settings needed to authenticate against the TCC service.</description>
+			</parameter-group>
+			<parameter name="username" type="text" required="true" groupName="auth">
+				<label>Username</label>
+				<description>Your TCC Username</description>
+				<default></default>
+			</parameter>
+			<parameter name="password" type="text" required="true" groupName="auth">
+				<label>Password</label>
+				<description>Your TCC Password</description>
+				<context>password</context>
+				<default></default>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" required="false" min="15" max="3000">
+				<label>Refresh Interval</label>
+				<description>The refresh interval to poll TCC (in seconds).</description>
+				<default>15</default>
+				<advanced>true</advanced>
+			</parameter>
+			<parameter name="applicationId" type="text" required="true">
+				<label>Application Id</label>
+				<description>The application id to use (probably won't change).</description>
```

Comment:
Well, this ID has been stable since I've started working on it. I'm doubting if it will ever change, but if it does, people could change it here. That way the binding wouldn't need an update. That said, I'm doubtful if it'll ever change. I can drop the statement in parenthesis maybe?

Reply:
But is this a private ID only for you device or do you also share it with all other users?

Reply:
It's the same for all users of this binding.

Reply:
Then please remove it :-)